client/app.json

```diff
@@ -1,13 +1,38 @@
 {
   "pages":[
     "pages/index/index",
-    "pages/addCgi/addCgi"
+    "pages/userinfo/userinfo",
+    "pages/addCgi/addCgi",
+    "pages/photo/photo",
+    "pages/editntc/editntc"
   ],
   "window":{
-    "backgroundColor":"#F6F6F6",
+    "backgroundColor":"#73a96b",
     "backgroundTextStyle":"light",
-    "navigationBarBackgroundColor": "#F6F6F6",
-    "navigationBarTitleText": "Wafer Quick Start",
+    "navigationBarBackgroundColor": "#73a96b",
+    "navigationBarTitleText": "TPCC YEC Soccer Team",
     "navigationBarTextStyle":"black"
+  },
+  "tabBar": {
+    "list": [
+      {
+        "pagePath": "pages/index/index",
+        "text": "通知",
```

Comment:
Please use English.

Reply:
Okay will change this

Reply:
Both sound good to me. So please use "Home" as it's shorter :)

Reply:
Sure :>

Reply:
By the way, do we need to make the page name to be consistent with the name in the tabbar?
For example, now, the third page in the tabbar is called "Me" but the page name is "userinfo", do we need to change this?

Reply:
I think that's fine.

Reply:
Okay, I have made the changes, except the home page (still called index page)

Reply:
Please check that, thanks!